.circleci/config.yml

```diff
@@ -48,6 +51,21 @@ jobs:
           command: |
             npm config set "//registry.npmjs.org/:_authToken=$NPM_TOKEN"
             npm publish
+  
+  prerelease:
+    <<: *container
+    steps:
+      - checkout
+      - *restore_deps
+      - run: npm install --ignore-scripts
+      - run:
+          name: Build the USWDS package
+          command: npm run release
+      - run:
+          name: Publish to NPM
+          command: |
+            npm config set "//registry.npmjs.org/:_authToken=$NPM_TOKEN"
+            npm publish . --tag alpha
```

Comment:
It seems like we just push the whole repo up to npm, since the `.zip` file is just put into the `/dist` folder, so I think this should be ok? We won't really know till we push it live 🌈